runtime/doc/develop.txt

```diff
@@ -34,7 +34,7 @@ balance must be found between them.
 VIM IS... VI COMPATIBLE					*design-compatible*
 
 First of all, it should be possible to use Vim as a drop-in replacement for
-Vi.  When the user wants to, he can use Vim in compatible mode and hardly
+Vi.  When the user wants to, they can use Vim in compatible mode and hardly
```

Comment:
I'm not a native English speaker, but "he" was referring the "the user" in the first part of the sentence.
So saying "they" does not look correct.

Same remark in some other places.
I don't think we need to workaround the fact that English has no neutral he/she pronoun.

Reply:
It is perfectly correct to use "they" here. In fact it is more common to write "they" in this situation, as until this point the gender has not been specified.

Reply:
Regardless of how much people push for it, neutral 'they' doesn't feel natural when not used as a wildcard.